README.md

```diff
@@ -4,7 +4,9 @@
     <h2>Frappe Bench</h2>
 </div>
 
-[![Build Status](https://travis-ci.org/frappe/bench.svg?branch=master)](https://travis-ci.org/frappe/bench)
+**Circle CI:** [![Circle CI](https://circleci.com/gh/frappe/bench.svg?style=svg)](https://circleci.com/gh/frappe/bench)
+
+**Travis CI:** [![Travis CI](https://travis-ci.org/frappe/bench.svg?branch=master)](https://travis-ci.org/frappe/bench)
```

Comment:
Remove this line. Not needed, as we shall discontinue with Travis for now.

Reply:
fixed.